CHANGELOG.md

```diff
@@ -7,205 +7,33 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
-- General backgrounds and borders design token updates ([#13764](https://github.com/MetaMask/metamask-extension/pull/13764))
-- Add `IconEye` and `IconEyeSlash` icon components ([#13978](https://github.com/MetaMask/metamask-extension/pull/13978))
-- Bump controllers to 27 ([#13985](https://github.com/MetaMask/metamask-extension/pull/13985))
-- Add `data-test-id` prop to `TextField` component ([#13973](https://github.com/MetaMask/metamask-extension/pull/13973))
-- Dark Mode: Select Account Hardware Wallets ([#13917](https://github.com/MetaMask/metamask-extension/pull/13917))
-- Update package.json ([#13961](https://github.com/MetaMask/metamask-extension/pull/13961))
-- E2e metrics ([#13904](https://github.com/MetaMask/metamask-extension/pull/13904))
+- Improving settings toggle accessibility by allowing label interaction ([#13876](https://github.com/MetaMask/metamask-extension/pull/13876))
 - Updating account and network icons ([#13947](https://github.com/MetaMask/metamask-extension/pull/13947))
-- Dark Mode: Update transaction detail activity log ([#13919](https://github.com/MetaMask/metamask-extension/pull/13919))
-- Automate the Flask release process ([#13898](https://github.com/MetaMask/metamask-extension/pull/13898))
-- Add labels to provide better accessibility during onboarding v2 ([#13890](https://github.com/MetaMask/metamask-extension/pull/13890))
-- Dark Mode: Fix currently selected network color ([#13918](https://github.com/MetaMask/metamask-extension/pull/13918))
-- Fix yarn.lock ([#13941](https://github.com/MetaMask/metamask-extension/pull/13941))
-- Permissions: convert hook to util ([#13906](https://github.com/MetaMask/metamask-extension/pull/13906))
-- Remove usages of useMetricEvent hook ([#13930](https://github.com/MetaMask/metamask-extension/pull/13930))
-- Show snap version on install ([#13931](https://github.com/MetaMask/metamask-extension/pull/13931))
+
+### Fixed
+- Add 'Enhanced Gas UI' setting to search index ([#14206](https://github.com/MetaMask/metamask-extension/pull/14206))
+- Fixed issue where the "lock screen" shows up briefly between the "create password" screen and the success screen ([#14111](https://github.com/MetaMask/metamask-extension/pull/14111))
+- Add space between sentences in insufficient currency warning ([#14019](https://github.com/MetaMask/metamask-extension/pull/14019))
+- Fix issue where editing advanced gas while speeeding up a transaction exits speedup ([#14101](https://github.com/MetaMask/metamask-extension/pull/14101))
+- Fix typo in cancel/speed up messaging ([#14067](https://github.com/MetaMask/metamask-extension/pull/14067))
+- Fix token icon when going from token detail page to Swaps view ([#14062](https://github.com/MetaMask/metamask-extension/pull/14062))
+- Fix missing insufficient funds error for token sends ([#14026](https://github.com/MetaMask/metamask-extension/pull/14026))
```

Comment:
`Fix missing insufficient funds error for token sends` was part of a previous release